compose/config/config_schema_v3.0.json

```diff
@@ -205,12 +205,13 @@
         "interval": {"type":"string"},
         "timeout": {"type":"string"},
         "retries": {"type": "number"},
-        "command": {
+        "test": {
           "oneOf": [
             {"type": "string"},
             {"type": "array", "items": {"type": "string"}}
           ]
-        }
+        },
+        "disable": {"type": "boolean"}
```

Comment:
Also, if we're adding this option, I don't think we need to support the `null` type for the healthcheck section anymore.